Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -1145,7 +1145,9 @@ inline void BuilderCustom::applyValueCursor(BuilderState& builderState, CSSValue
     auto& list = downcast<CSSValueList>(value);
     for (auto& item : list) {
         if (auto* image = dynamicDowncast<CSSCursorImageValue>(item)) {
-            builderState.style().addCursor(builderState.createStyleImage(*image), image->hotSpot());
+            auto styleImage = image->createStyleImage(builderState);
+            auto hotSpot = styleImage->hotSpot();
+            builderState.style().addCursor(WTFMove(styleImage), WTFMove(hotSpot));
```

Comment:
I don’t think we need to `WTFMove` a `std::optional<IntPoint>`. It does no harm, but for brevity maybe not?